composer.lock

```diff
@@ -961,5 +1493,6 @@
     "prefer-stable": false,
     "prefer-lowest": false,
     "platform": [],
-    "platform-dev": []
+    "platform-dev": [],
+    "plugin-api-version": "1.1.0"
 }
```

Comment:
There were no changes to `composer.json`, why should the lockfile be updated?

Reply:
This was done to address a Circle CI failure with the message:

```
Your lock file does not contain a compatible set of packages. Please run composer update.
```